.kokoro/tests/run_tests.sh

```diff
@@ -37,9 +37,15 @@ cd github/python-docs-samples
 # install nox for testing
 pip install -q nox
 
-# Unencrypt and extract secrets
-SECRETS_PASSWORD=$(cat "${KOKORO_GFILE_DIR}/secrets-password.txt")
-./scripts/decrypt-secrets.sh "${SECRETS_PASSWORD}"
+# update gcloud version
+# TODO: move this to the docker image
+gcloud components update
+
+# Use secrets acessor service account to get secrets
+gcloud auth activate-service-account \
```

Comment:
Does it make sense to just use ${GOOGLE_APPLICATION_CREDENTIALS}? We can give it the right permission on the target project I think.

Reply:
The scripts are using secrets manager via gcloud, so I think we need to use `gcloud auth activate-service-account`.  I thought only the libraries used `GOOGLE_APPLICATION_CREDENTIALS`?

Reply:
We run `activate-service-account` few lines below. It's for using cloudbuild from cloud-run tests.

Reply:
Ah I see. `testing/service-account.json` doesn't exist until the `scripts/decrypt-secrets.sh` has been executed though.

I could move the python-docs-samples-testing service account into `KOKORO_GFILE_DIR` (or keystore)?

Reply:
Oh I see. Then it's fine I guess.